subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/AggregatingIncrementalAnnotationProcessingIntegrationTest.groovy

```diff
@@ -178,17 +178,36 @@ class AggregatingIncrementalAnnotationProcessingIntegrationTest extends Abstract
         outputs.deletedClasses("ServiceRegistry")
     }
 
-    def "processors can't access resources"() {
+    def "processors can't read resources"() {
         given:
-        withProcessor(new NonIncrementalProcessorFixture().withDeclaredType(IncrementalAnnotationProcessorType.AGGREGATING))
-        java "@Thing class A {}"
+        withProcessor(new NonIncrementalProcessorFixture(false, true, false).withDeclaredType(IncrementalAnnotationProcessorType.AGGREGATING))
+        def a = java "@Thing class A {}"
+        outputs.snapshot { succeeds"compileJava" }
 
-        expect:
-        fails "compileJava"
+        when:
+        a.text = "@Thing class A { void foo() {} }"
+
+        then:
+        succeeds"compileJava", "--info"
```

Comment:
Tiny formatting issue: These could use a space after `succeeds` (same below)

Reply:
Formatted the whole file